specification/marketplacenotifications/resource-manager/Microsoft.MarketplaceNotifications/stable/2021-03-03/MarketplaceNotifications.json

```diff
@@ -0,0 +1,256 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Blueprint API",
```

Comment:
Are you from the Marketplace team or have you gotten explicit signoff from the marketplace team to use the Microsoft.MarketplaceNotifications namespace?

Reply:
yes, we are part of Marketplace team